backend/Task.py

```diff
@@ -0,0 +1,34 @@
+from flask import jsonify
+class Task():
+
+    def __init__(self, taskId, userId, name, desc, clas, dueDate, targetDate, remind, remindDate=None):
+        self.taskId = taskId
+        self.userId = userId
+        self.name = name
+        self.desc = desc
+        self.clas = clas
+        self.dueDate = dueDate
+        self.targetDate = targetDate
+        self.remind = remind
```

Comment:
Nit: Having a remindDate should be enough, the extra remind bool is unnecessary because remindDate will be None if the user didn't provide one.

Reply:
Good point, we can drop it

Reply:
I thought that was unnecessary too.